Rocket.Chat/Controllers/Chat/ChatControllerMessageActions.swift

```diff
@@ -71,9 +67,26 @@ extension ChatViewController {
             self?.reply(to: message, onlyQuote: true)
         })
 
-        var actions = [react, pin, report, copy, reply, quote]
+        var actions = [react, reply, quote, copy, report]
+
+        if auth.canPinMessage(message) == .allowed {
+            let pinMessage = message.pinned ? localized("chat.message.actions.unpin") : localized("chat.message.actions.pin")
+            let pin = UIAlertAction(title: pinMessage, style: .default, handler: { (_) in
+                if message.pinned {
+                    MessageManager.unpin(message, completion: { (_) in
+                        // Do nothing
```

Comment:
@rafaelks Couldn't we make `completion` optional and make its default value `nil`? This way we avoid empty closures like this one.